quokka_editor_front/src/index.css

```diff
@@ -14,11 +14,11 @@
 
 /* Handle */
 ::-webkit-scrollbar-thumb {
-  background: #e7c8a0;
+  background: #5d5f6d;
   border-radius: 5px;
 }
 
 /* Handle on hover */
 ::-webkit-scrollbar-thumb:hover {
-  background: #e7c8a0;
+  background: #5d5f6d;
 }
```

Comment:
You can extract bg color to css variable. Define the set of app colours and use them